javav2/usecases/creating_fsa_app/src/test/java/FSATests.java

```diff
@@ -0,0 +1,97 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+import com.example.fsa.services.DetectSentimentService;
+import com.example.fsa.services.ExtractTextService;
+import com.example.fsa.services.PollyService;
+import com.example.fsa.services.S3Service;
+import com.example.fsa.services.TranslateService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestMethodOrder;
+import java.io.IOException;
+import java.io.InputStream;
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class FSATests {
+
+    // Specify the key to use. 
+    private static final String objectName = "Enter the object name here";
+    
+    // Specify the bucket name.
+    private static final String bucketName = "Enter the bucket name here";
+
+    private ExtractTextService textService;
+    private TranslateService translateService;
+    private PollyService pollyService;
+    private S3Service s3Service;
+    private DetectSentimentService sentimentService;
+
+    @BeforeEach
+    public void setUp() {
+        textService = new ExtractTextService();
+        translateService = new TranslateService();
+        pollyService = new PollyService();
+        s3Service = new S3Service();
+        sentimentService = new DetectSentimentService();
+    }
+
+    @Test
+    @Tag("IntegrationTest")
+    @Order(1)
+    public void testTextExtraction() {
+        String text = textService.getCardText(bucketName, objectName);
```

Comment:
Is this text expecting a bucket with the object to exist? Where is that set up?

Reply:
For now - I am referencing static resources. I will look into changing that way of testing down the road.